docs/mkdocs/docs/api/macros/nlohmann_json_namespace_begin.md

```diff
@@ -1,38 +1,47 @@
 # NLOHMANN_JSON_NAMESPACE_BEGIN, NLOHMANN_JSON_NAMESPACE_END
 
 ```cpp
-#define NLOHMANN_JSON_NAMESPACE_BEGIN  // (1)
-#define NLOHMANN_JSON_NAMESPACE_END    // (2)
+#define NLOHMANN_JSON_NAMESPACE_BEGIN /* value */  // (1)
+#define NLOHMANN_JSON_NAMESPACE_END   /* value */  // (2)
 ```
 
-These macros can be used to open and close the `nlohmann` namespace. They include an inline namespace used to
-differentiate symbols when linking multiple versions (including different ABI-affecting macros) of this library.
+These macros can be used to open and close the `nlohmann` namespace. They include two inline namespaces used to
```

Comment:
Does "They" stand for "These macros"? As in

> These macros include two inline namespaces

? This confuses me.

Reply:
Yes, as in the opening macro contains two `inline namespace <NAME> {` parts and the closing macro the matching curly brackets. I may have to rewrite it anyway, let's see.